doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -123,6 +123,11 @@ shows each existential with the hypotheses and conclusion for the associated goa
 
    Show Existentials.
 
+Users can control which goals are displayed in the context by :term:`focusing <focus>` and
+_shelving_ goals.  Focusing lets the user pick a single goal to work on.  Focusing operations
```

Comment:
You probably meant to use italic here. That would be with `*` instead of `_`.

Reply:
They are very often not useful. I call on your personal judgment to decide when to keep them or not.